### ISSUE TITLE
Pagination: invalid ARIA attribute `ariaLabel`

### ISSUE THREAD

**Issue by AndreiEnache** [Semantic-Org/Semantic-UI-React#2601]
### Steps
Create a small pagination component.
When the component mounts, a warning is displayed.

### Expected Result
Not show the warning.

### Actual Result
Shows the warning.

### Version
* semantic-ui-react: 0.78.3
* react: 16.2.0
* react-dom: 16.2.0

### Testcase
https://codesandbox.io/s/9l90v9xnwp

**Comment by brianespinosa** [comment]
Valid bug. I do not think this is a duplicate. PRs welcome.

**Comment by tphan18** [comment]
Confirmed. This is a valid bug.

**Comment by gtod** [comment]
I think the problem is that the `arialLabel` prop on `PaginationItem` is being copied into the `MenuItem` *in addition* to the correct `aria-label` prop.

I say this only because if you replace the current `MenuItem.create` line [here](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/addons/Pagination/PaginationItem.js#L79) with the following 3 lines the error goes away:
```js
  var props = Object.assign({}, this.props);
  delete props.ariaLabel;
  return MenuItem.create(props, {
```

**Comment by brianespinosa** [comment]
@gtod there is already a PR being worked on for this.  🎉 Please see above history for reference to #2607

**Comment by tphan18** [comment]
I don't think the issue is fixed in v0.79.0.

**Comment by layershifter** [comment]
Create a new issue with repro on codesandbox, please.
